packages/components/src/internal/components/samples/actions.ts

```diff
@@ -938,5 +938,5 @@ export async function getFieldLookupFromSelection(
         }
     }
 
-    return [...sampleIds];
+    return Array.from(sampleIds);
```

Comment:
Is this just preferred syntax or is there a reason to prefer this?

Reply:
I probably didn't need to leave this in.

Reply:
Removed.